Doctrine/Phpcr/SitemapDocumentProvider.php

```diff
@@ -42,7 +42,7 @@ public function load($sitemap)
     {
         // todo rewrite query when https://github.com/symfony-cmf/CoreBundle/issues/126 is ready
         $documentsCollection = $this->manager->createQuery(
-            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true)',
+            'SELECT * FROM [nt:unstructured] WHERE (visible_for_sitemap = true) ORDER By [jcr:path]',
```

Comment:
in general i expect there to be no guaranteed order without an order by
clause. the last section on this doc is rather confusing - but i think
its about a product not using the normal jackrabbit, so no idea how
relevant it is:

https://www.exoplatform.com/docs/public/index.jsp?topic=%2FPLF35%2FJCR.SearchingRepositoryContent.OrderingSpecifying.html

how did ORDER BY [jcr:path] DESC fail? invalid query or just different
html? the order of the output would be reverted if it works...